README.md

```diff
@@ -129,9 +125,16 @@ flutter_gen:
     flutter_svg: true
     flare_flutter: true
 
+  assets:
+    enabled: true
```

Comment:
These are optional and we have a default settings section in README.
How about adding a link to that section and omit these optional here?
As we can reduce some noise.